com.reprezen.swagedit.core/src/com/reprezen/swagedit/core/assist/JsonReferenceProposalProvider.java

```diff
@@ -112,66 +125,103 @@ public String value() {
         public String label() {
             return label;
         }
-        
+
+        public boolean isLocalOnly() {
+            return isLocalOnly;
+        }
+
+        /**
+         * Returns all proposals found in the document at the specified field.
+         * 
+         * @param document
+         * @param fieldName
+         * @param path
+         * @return Collection of proposals
+         */
+        public Collection<Proposal> collectProposals(JsonNode document, IPath path) {
+            final Collection<Proposal> results = Lists.newArrayList();
+            if (value() == null) {
+                return results;
+            }
+
+            final JsonNode parameters = ValidationUtil.findNode(value(), document);
```

Comment:
We name the proposal items `parameters` since the times when we only had code assist for parameters. It's time to rename it :)